game/screen_songs.cc

```diff
@@ -356,7 +356,8 @@ void ScreenSongs::draw() {
 std::string ScreenSongs::getHighScoreText() const {
 	auto const scores = m_database.queryPerSongHiscore(m_songs.currentPtr());
 	auto const datetimeFormat = config["game/datetime_format"].so();
-	auto const maxLines = 8;
+	// allows for example for 16 entries for a single track
+	auto const maxLines = 17;
```

Comment:
Did you try it and does all lines are displayed correct on the screen? (Use window mode and different windows sizes)

Cause layout rendering is not that good in performous.